templates/site/js/main.js

```diff
@@ -267,105 +209,65 @@ function deleteNodeUser(username, nodeName){
             }).then(response => response.json()).then(json =>{
 
                     if(json.code == 200){
-                        sessionStorage.clear();
```

Comment:
This now does nothing? We need to report some sort of feedback to the user if this is successful (and also if it's not successful)

Reply:
This is a placeholder function.  The feedback isn't in the design yet. I would imagine the user experience would be:

- User clicks delete button
- User account is deleted from the particular node
- User sees an indication of some kind that the delete has happened
- (Node Network popup is still visible, perhaps)